docs/rcac.md

```diff
@@ -0,0 +1,32 @@
+# nf-core/configs: Purdue RCAC Configuration
+
+nf-core pipelines have been configured for use on the ACCESS Anvil and community clusters operated by Rosen Center for Advanced Computing (RCAC) at Purdue University.
+
+To use RCAC's profile, run the pipeline with `-profile rcac`. 
+
+Example: `nextflow run -profile rcac`
+
+Users can also put the `nextflow ...` command into a batch script and submit the job to computing nodes by `sbatch` or launch interative jobs to computing nodes by `sinteractive`. Using this way, both nextflow manager processes and tasks will run on the allocated compute nodes using the `local` executor. It is recommended to use `-profile singularity`
```

Comment:
```suggestion
Users can also put the `nextflow ...` command into a batch script and submit the job to computing nodes by `sbatch` or launch interative jobs to computing nodes by `sinteractive`. Using this way, both Nextflow manager processes and tasks will run on the allocated compute nodes using the `local` executor. It is recommended to use `-profile singularity`
```

Reply:
If you recommend using `singularity`, I would actually specify this in teh config itself